docs/api/clojure/symbol.md

```diff
@@ -0,0 +1,150 @@
+            [org.apache.clojure-mxnet.symbol :as sym]
+            [org.apache.clojure-mxnet.context :as context]))
+```
+
+
+## How to Compose Symbols
+
+The symbolic API provides a way to configure computation graphs.
+You can configure the graphs either at the level of neural network layer operations or as fine-grained operations.
+
+The following example configures a two-layer neural network.
+
+```clojure
+(def data (sym/variable "data"))
+(def fc1 (sym/fully-connected "fc1" {:data data :num-hidden 128}))
+(def act1 (sym/activation "act1" {:data fc1 :act-type "relu"}))
+(def fc2 (sym/fully-connected "fc2" {:data act1 :num-hidden 64}))
+(def net (sym/softmax-output "out" {:data fc2}))
+```
+
+You could also combine this more dynamically with:
+
+```clojure
+(as-> (sym/variable "data") data
+  (sym/fully-connected "fc1" {:data data :num-hidden 128})
+  (sym/activation "act1" {:data data :act-type "relu"})
+  (sym/fully-connected "fc2" {:data data :num-hidden 64})
+  (sym/softmax-output "out" {:data data}))
+
+net ;=> #object[org.apache.mxnet.Symbol 0x5c78c8c2 "org.apache.mxnet.Symbol@5c78c8c2"] 
+```
+
+The basic arithmetic operators (plus, minus, div, multiplication) work as expected.
+
+The following example creates a computation graph that adds two inputs together.
+
+```clojure
+(def a (sym/variable "a"))
+(def b (sym/variable "b"))
+(def c (sym/+ a b))
+```
+
+## More Complicated Compositions
+
+MXNet provides well-optimized symbols for layers commonly used in deep learning (see src/operator). We can also define new operators in Python. The following example first performs an element-wise add between two symbols, then feeds them to the fully connected operator:
+
+```clojure
+(def lhs (sym/variable "data1"))
+(def rhs (sym/variable "data2"))
+(def net (sym/fully-connected "fc1" {:data (sym/+ lhs rhs) :num-hidden 128}))
+(sym/list-arguments net) ;=> ["data1" "data2" "fc1_weight" "fc1_bias"]
+```
+
+## Group Multiple Symbols
+
+To construct neural networks with multiple loss layers, we can use mxnet.sym.Group to group multiple symbols together. The following example groups two outputs:
+
+```clojure
+(def net (sym/variable "data"))
+(def fc1 (sym/fully-connected {:data net :num-hidden 128}))
+(def net2 (sym/activation {:data fc1 :act-type "relu"}))
+(def out1 (sym/softmax-output {:data net2}))
+(def out2 (sym/linear-regression-output {:data net2}))
+(def group (sym/group [out1 out2]))
+(sym/list-outputs group)
+;=> ["softmaxoutput0_output" "linearregressionoutput0_output"]
+```
+
+## Serialization
+ There are two ways to save and load the symbols. You can use the mxnet.Symbol.save and mxnet.Symbol.load functions to serialize the Symbol objects. The advantage of using save and load functions is that it is language agnostic and cloud friendly. The symbol is saved in JSON format. You can also get a JSON string directly using mxnet.Symbol.toJson. Refer to API documentation for more details.
```

Comment:
two ways.... `mxnet.Symbol.save` and `mxnet.Symbol.load`? (but that's really one way isn't it? What's the other way to save and load?)

Also, can you link to the docs?